src/docs/apiExamples/SankeyChart.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  Sankey,
+  Tooltip,
+  Layer,
+  Rectangle,
+} from 'recharts';
+
+const data0 = {
+  nodes: [
+    { name: 'Visit' },
+    { name: 'Direct-Favourite' },
+    { name: 'Page-Click' },
+    { name: 'Detail-Favourite' },
+    { name: 'Lost' },
+  ],
+  links: [
+    { source: 0, target: 1, value: 3728.3 },
+    { source: 0, target: 2, value: 354170 },
+    { source: 2, target: 3, value: 62429 },
+    { source: 2, target: 4, value: 291741 },
+  ],
+};
+
+const MyCustomNode = ({
+  x,
+  y,
+  width,
+  height,
+  index,
+  payload,
+  containerWidth,
+}) => {
+  const isOut = x + width + 6 > containerWidth;
+  return (
+    <Layer key={`CustomNode${index}`}>
+      <Rectangle
+        x={x}
+        y={y}
+        width={width}
+        height={height}
+        fill="#5192ca"
+        fillOpacity="1"
+      />
+      <text
+        textAnchor={isOut ? 'end' : 'start'}
+        x={isOut ? x - 6 : x + width + 6}
+        y={y + height / 2}
+        fontSize="14"
+        stroke="#333"
+      >
+        { payload.name }
+      </text>
+      <text
+        textAnchor={isOut ? 'end' : 'start'}
+        x={isOut ? x - 6 : x + width + 6}
+        y={y + height / 2 + 13}
+        fontSize="12"
+        stroke="#333"
+        strokeOpacity="0.5"
+      >
+        {`${payload.value}k` }
+      </text>
+    </Layer>
+  );
+};
```

Comment:
I think this would be good to add to the docs for the SankeyChart. I couldn't figure out how to add labels to the nodes until I found this.

Reply:
Also think it's worth adding!